packages/api-client-core/spec/InternalModelManager.spec.ts

```diff
@@ -134,13 +134,23 @@ describe("InternalModelManager", () => {
       const plan = internalFindManyQuery("widget", undefined);
       expect(plan).toMatchInlineSnapshot(`
         {
-          "query": "query InternalFindManyWidget ($after: String, $before: String, $first: Int, $last: Int) { internal  { listWidget (after: $after, before: $before, first: $first, last: $last) { pageInfo { hasNextPage, hasPreviousPage, startCursor, endCursor }, edges { cursor, node } } } }",
```

Comment:
Lots of speculation on this one... but I'm not sure whether you are doing query parsing caching at all but previously it would always generate the same query (with unused variables) and therefore (if cached) would hit the same cache key however now there are more distinct queries being generated depending on variables provided.